File: Web Design and UI/7. SPA/ASP.NET SOAP To RESTful Converter1/ASP.NET SOAP To RESTful Converter1/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace ASP.NET_SOAP_To_RESTful_Converter1.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewBag.Title = "Home Page";

            return View();
        }
    }
}

File: Web Design and UI/7. SPA/TheBigCatProjectClient/TheBigCatProject.Client/scripts/app/home/home-controller.js
(function () {
    'use strict';

    function HomeController(cats) {
        var vm = this;

        cats.searchCats({ page: 1 })
        .then(function (initialsCats) {
            vm.cats = initialsCats;
        })
    }

    angular.module('catApp.controllers')
    .controller('HomeController', ['cats', HomeController]);
}());